packages/emitter-framework/src/csharp/components/utils/doc-comments.tsx

```diff
@@ -0,0 +1,60 @@
+import * as ay from "@alloy-js/core";
+import * as cs from "@alloy-js/csharp";
+import { Type } from "@typespec/compiler";
+import { Typekit } from "@typespec/compiler/typekit";
+
+/**
+ * Helper to render a doc string for a given TypeSpec type.
+ *
+ * This is not a JSX component as it needs to return undefined if there is no doc.
+ *
+ * @param $ The Typekit instance
+ * @param type The TypeSpec type to generate documentation for
+ * @returns A DocSummary component containing the rendered doc string, or undefined if no doc is available.
+ */
+export function getDocComments($: Typekit, type: Type): ay.Children {
+  const typeDoc = $.type.getDoc(type);
+  if (!typeDoc) {
+    return undefined;
```

Comment:
Thoughts on bailing early if there's no docs (even if there are parameters with docs)?

Feels weird to have a doc-comment with no `<summary>`